pvr.wmc/changelog.txt

```diff
@@ -1,4 +1,7 @@
-﻿2.4.0
+﻿2.4.5
```

Comment:
2.4.4

Reply:
I didn't think 2.4.4 needed to be there since it was the same as 2.4.5 and very few people used 2.4.4.